projects/demo/src/modules/app/home/home.template.html

```diff
@@ -214,9 +214,9 @@
                 <code>SVG</code> inlining. Taiga UI icons will be automatically
                 downloaded and inlined directly into DOM. However, Angular
                 <code>Sanitizer</code> removes <code>SVG</code> code in that
-                case. To use custom sanitizer that properly supports
-                <code>SVG</code> provide <code>TUI_SANITIZER</code> token. We
-                recommend you using our library
+                case. To use a custom sanitizer that properly supports
+                <code>SVG</code> provide the <code>TUI_SANITIZER</code> token. We
+                recommend you, using our library
```

Comment:
I think this should rather be changed to `...you to use our...`
Comma seems wrong here.

Reply:
I was in between your suggestion and what I wrote. I will do the change :)

Reply:
@waterplea I did the changes